### DISCUSSION TITLE
Upload existing annotations?

### DISCUSSION THREAD

**Discussion by nikola-jensen** [biigle/core#962] [Q&A]
Hello! I have been running an object detection model locally on my computer (which outputs yolov9 formatted annotations), and would like to upload the results to biigle to review them. Is this a feature that is supported at this time? 
Thanks!

**Comment by mzur** [comment]
Hi! On biigle.de we have recently implemented support for importing annotations in the COCO format. So if you manage to convert your YOLO file to COCO, you can import the annotations via the user interface. If not, you can still upload the annotations via the API. There is an [example with pascalVOC annotations](https://github.com/biigle/community-resources/blob/7fc611761349c8ebf986aa8c6b31083265e45f96/annotations_to_biigle/annotations_to_biigle.py#L29-L36) in the BIIGLE community scripts.
